apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -45,6 +45,15 @@ provide a general-purpose abstraction over raw the RPC API, providing
 features such as page streaming to make client code cleaner and
 simpler.
 
+[SimplePublisher](obj/api/Google.Cloud.PubSub.V1.SimplePublisher.yml) and [SimpleSubscriber](obj/api/Google.Cloud.PubSub.V1.SimpleSubscriber.yml) provide simpler APIs for message publishing and subscribing.
```

Comment:
I think you can be bolder here, talking about efficiencies etc. (Or do that later...)